examples/MomentoRedisLambdaExample/MomentoRedisExampleLambdaHandler/src/MomentoRedisExampleLambdaHandler/Function.cs

```diff
@@ -0,0 +1,300 @@
+using Amazon.Lambda.Core;
+using StackExchange.Redis;
+using Momento.Sdk;
+using Momento.Sdk.Auth;
+using Momento.Sdk.Config;
+using Momento.Sdk.Responses;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace MomentoRedisExampleLambdaHandler;
+
+public class Input
+{
+    public bool RunMomentoWithCompression { get; set; }
+    public bool RunMomentoCompressionWithSleep { get; set; } 
+    public bool RunMomentoWithoutCompression { get; set; }
+    public bool RunRedisWithCompression { get; set; }
+    public bool RunRedisWithoutCompression { get; set; }
```

Comment:
Can these be narrowed down to eliminate the ones that are opposite one another? You should be able to assume that `RunMomentoWithCompression == false` is equivalent to `RunMomentoWithoutCompression` and avoid some of the input validation below. `RunMomentoWithCompressionWithSleep` could also be shortened to `CompressWithSleep` since it can just be ignored if `RunMomentoWithCompression` is false.